showcase/gapic-showcase/src/test/java/com/google/showcase/v1beta1/it/ITAutoPopulatedFields.java

```diff
@@ -120,8 +120,8 @@ public void sendMessage(ReqT message) {
   private EchoClient httpJsonClient;
   private EchoClient httpJsonClientWithRetries;
 
-  @Before
-  public void createClients() throws Exception {
+  @BeforeEach
```

Comment:
Same with this. I know it isn't a test you wrote, but could we try with @BeforeAll? If it doesn't work, let's keep it as is.